addons/resource/models/resource_resource.py

```diff
@@ -34,6 +34,11 @@ def default_get(self, fields):
         ('material', 'Material')], string='Type',
         default='user', required=True)
     user_id = fields.Many2one('res.users', string='User', help='Related user name for the resource to manage its access.')
+    avatar_128 = fields.Image(compute='_compute_avatar_128')
+    im_status = fields.Char(related='user_id.im_status')
+    share = fields.Boolean(related='user_id.share')
+    email = fields.Char(related='user_id.email')
+    phone = fields.Char(related='user_id.phone')
```

Comment:
> I am wondering if we should not fetch those data in your loadAdditionalData to avoid adding many related fields just to be able to use the avatar popover? 🤔

As discussed, the idea was to have a generic component (as much as model-independent as possible) for the avatar card. So keeping those related fields is the best option and ok for hr as they are not stored.